File: server/dotnet/sdk-dotnet-example/Business/Domain/Payments/Mandate.cs
using Business.Domain.Common;
using Business.Domain.Enums.Payments;

namespace Business.Domain.Payments;

public class Mandate
{
    public string? IBAN { get; set; }
    
    public string? CustomerReference { get; set; }
    
    public string? MandateReference { get; set; }

    public RecurrenceType? RecurrenceType { get; set; }

    public SignatureType? SignatureType { get; set; }

    public string? ReturnUrl { get; set; }

    public AddressDto? Address { get; set; }
}

File: server/nodejs/src/business/domain/payments/mandate.ts
import { RecurrenceType } from '../enums/payments/recurrence-type';
import { AddressDto } from '../../dtos/common/address-dto';
import { SignatureType } from '../enums/payments/signature-type';

export type Mandate = {
    iban?: string | null;
    customerReference?: string | null;
    mandateReference?: string | null;
    recurrenceType?: RecurrenceType | null;
    signatureType?: SignatureType | null;
    returnUrl?: string | null;
    address?: AddressDto | null;
};
